frontend/client/src/components/ChangeRoleModal.js

```diff
@@ -16,7 +16,7 @@ const ChangeRoleModal = withStore((props) => {
 
   return (
     <Modal hidden={props.hidden} onClose={props.onClose} containerClass="change-role-modal-container">
-      <h2>Confirm Change</h2>
+      <h3>Confirm Change</h3>
```

Comment:
If we are using `<h3>` for modal titles, let's use that consistently across all modals.  So for example line 106 of `ChangePasswordModal.js` I believe is using an `<h2>`, when it should be an `<h3>`

Reply:
I agree with this, all modal headers should be `<h3>`